### PR TITLE
Creates new login timers

### PR DESCRIPTION
**Pull request by daedela**
Creates a 5 minute timer to create a name which refreshes on prompt.
Creates a 30 second timer to create a password which also refreshes on prompt.

### CODE REVIEW

File: src/interpreter.cpp [hunk 1]
```diff
@@ -2344,25 +2351,29 @@ void nanny(DescriptorData *d, char *arg) {
...
+                if (!PLR_FLAGGED(d->character, PLR_LOGIN))
+                    SET_FLAG(PLR_FLAGS(d->character), PLR_LOGIN);
+                event_create(EVENT_PASS_TIMEOUT, pass_timeout, d, false, nullptr, PASS_TIMEOUT);
+                REMOVE_FLAG(PLR_FLAGS(d->character), PLR_LOGIN);
             }
         } else {
             /* No player loaded: we might make a new character. */
 
-            if (ispell_name_check(tmp_name)) {
+//            if (ispell_name_check(tmp_name)) {
```
Comment (stridera) [inline src/interpreter.cpp:2362]: We're no longer instantly rejecting names in the dictionary?  (I approve.)

**Comment by daedela** [comment]
Oh crap, I forgot I have that line commented out to make the game run on my
home instance.  Ask Chinok though.

File: include/interpreter.hpp [hunk 2]
```diff
@@ -90,7 +90,9 @@ struct XName {
     char name[MAX_NAME_LENGTH + 3];
     XName *next;
 };
-#define NAME_TIMEOUT 1 * (30 RL_SEC) /* 5 minutes */
+#define NAME_APPROVE_TIMEOUT 30 RL_SEC
+#define NAME_TIMEOUT 300 RL_SEC /* 5 minutes */
```
[the remark this answers is not in the document]
Comment (daedela) [reply include/interpreter.hpp]: Updated from 5 mins to 1 min

File: src/interpreter.cpp (hunk 1 above)
Comment (daedela) [reply src/interpreter.cpp]: Removed the commenting out - my bad!

**Review by stridera — APPROVED** [review]
Might be more complex than required, but I think it'll do what you want.

**Comment by daedela** [comment]
So this was causing prod to crash, but I can't replicate on test or on my home instance.  Is there any way you can help me figure out what's going on?

**Comment by stridera** [comment]
Hard to tell.  You let the timeout happen?

**Comment by daedela** [comment]
Yep, I did it from all the affected screens too...

**Pull request merged**